packages/lab/src/date-picker/DatePickerHelperText.css

```diff
@@ -0,0 +1,8 @@
+.saltDatePickerHelperText {
+  visibility: visible;
+  margin-top: var(--salt-spacing-75);
+}
+
+.saltDatePickerHelperText-open {
+  visibility: hidden;
```

Comment:
What does hidden help here? A11y?

Reply:
I want to hide but not remove as that affects the layout.